stdlib/public/Concurrency/Task.swift

```diff
@@ -277,6 +292,8 @@ extension Task {
   /// Flags for schedulable jobs.
   ///
   /// This is a port of the C++ FlagSet.
+  /// ◊TR: What is FlagSet?
+  /// ◊TR: Why does that matter to users that it matches something in C++?
```

Comment:
This is an internal type, so we don't need to document it in depth for end users